markets/perps-market/contracts/modules/AsyncOrderViewModule.sol

```diff
@@ -0,0 +1,17 @@
+//SPDX-License-Identifier: MIT
+pragma solidity >=0.8.11 <0.9.0;
+
+import {IAsyncOrderViewModule} from "../interfaces/IAsyncOrderViewModule.sol";
+import {AsyncOrder} from "../storage/AsyncOrder.sol";
+import {PerpsMarket} from "../storage/PerpsMarket.sol";
+
+contract AsyncOrderViewModule is IAsyncOrderViewModule {
```

Comment:
thoughts on creating an `AsyncOrderSettlementModule` instead?  Would be similar to what we have in spot market.  

can move settle related items into the above module.

Reply:
Happy to align with spot market!

Reply:
Done